src/types/channelsData.ts

```diff
@@ -0,0 +1,12 @@
+export interface ChannelsData {
```

Comment:
It does not make sense separate channelsChartData.ts and channelsData.ts. It should be on file called channels.ts